fiftyone/plugins/__init__.py

```diff
@@ -1,45 +1,28 @@
 """
+FiftyOne plugins.
+
 | Copyright 2017-2023, Voxel51, Inc.
 | `voxel51.com <https://voxel51.com/>`_
 |
 """
 import types
-import deprecated
-import wrapt
 
 from .core import (
     enable_plugin,
     disable_plugin,
     delete_plugin,
+    list_plugins,
     list_disabled_plugins,
     list_downloaded_plugins,
     list_enabled_plugins,
     find_plugin,
     download_plugin,
     create_plugin,
 )
-from .definitions import list_plugins
-
-
-@deprecated.deprecated(reason="Use list_plugins() instead")
-def list_all_plugins():
-    """Wrapper for backwards compatibility. Returns a list of all plugins
-    that have not explicitly been disabled.
-
-    .. warning::
-
-            This method is deprecated and will be removed in a future release.
-            Use the drop-in replacement :meth:`list_plugins()` instead.
-    """
-
-    return list_plugins()
-
 
 # This enables Sphinx refs to directly use paths imported here
-# and ignore functions marked as depreciated
 __all__ = [
     k
     for k, v in globals().items()
-    if not k.startswith("_")
-    and not (type(v) in [wrapt.decorators.AdapterWrapper, types.ModuleType])
+    if not k.startswith("_") and not isinstance(v, types.ModuleType)
```

Comment:
I believe I had an issue with Sphinx including imported modules when using `isinstance` so I used `type`

Reply:
This same code block is inserted elsewhere in the repo, so I'm keeping it the same for consistency. 

FWIW the docs build fine for me.